src/formatDate.js

```diff
@@ -50,7 +50,82 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const newDate = [null, null, null];
+
+  const oldPosition = {
+    separator: fromFormat[3],
+
+    get date() {
+      return fromFormat.indexOf('DD');
+    },
+
+    get month() {
+      return fromFormat.indexOf('MM');
+    },
+
+    get year() {
+      return 3 - this.date - this.month;
```

Comment:
3 looks like a magic number here. Let's move to constants

Reply:
@naziamoff 👍 
Hi! 
I've added it to objects properties, please, check